### DISCUSSION TITLE
The blue 4330 and 4831 sensors move the thermistor next to the sensing foil which results in much improved performance of the temperature sensor

### DISCUSSION THREAD

**Discussion by nikosocean** [OceanGlidersCommunity/Oxygen_SOP#100] [Q&A] [answered]
Do we know how much this change improves the performance of the sensor? It is important for the quality of data?

**Comment by soerenthomsen** [comment, marked as answer]
Hi @nizaroka thanks,
I already created an issue here #98. These specific topics are best handles in specific issues.
